packages/core/src/components/badge/badge.scss

```
@@ -0,0 +1,427 @@
+  border: 0.0625rem solid var(--theme-color-1);
+}
+
+/* SR-only description slot (light DOM for aria-describedby). */
+:host ::slotted(.description) {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)